pkg/distro/fedora/package_sets.go

```diff
@@ -498,13 +497,7 @@ func iotInstallerPackageSet(t *imageType) rpmmd.PackageSet {
 	// include anaconda packages
 	ps := anacondaPackageSet(t)
 
-	releasever := t.Arch().Distro().Releasever()
-	version, err := strconv.Atoi(releasever)
-	if err != nil {
-		panic("cannot convert releasever to int: " + err.Error())
-	}
-
-	if version >= 38 {
+	if !common.VersionLessThan(t.arch.distro.osVersion, "39") {
```

Comment:
Just making sure that the intention here is to basically also change the condition to `version >= 39`, which is consistent with the previous commit and the rest of this file, but not mentioned explicitly in the commit message with this commit.

Reply:
It is the intention to move this to make it consistent with the other version checks which were all `>= 39`.